### DISCUSSION TITLE
After removing lockable flag from gitattributes, files are still read-only.

### DISCUSSION THREAD

**Discussion by andrea-cassioli-maersk** [git-lfs/git-lfs#4833] [Q&A]
Hi,
I have been experimenting with `git-lfs` and in particular I am looking at locking. I have added the `lockable` flag to a tracking entry in `.gitattributes` like 

```
*.jpg filter=lfs diff=lfs merge=lfs -text lockable
```

and it works well. Files are now read-only and I need lock/unlock to edit. 

However If I remove the flag I have noticed that while new files are editable, files that where tracked as lockable still are. They are also still read-only. Am I missing anything in this flow?

**Comment by bk2204** [comment]
Hey,

As outlined in #4484, it's not possible for the `post-commit` hook to know when a pattern has been removed from the file, so it's not possible for us to unlock them automatically if you edit the file manually.  You'd need to use the `git lfs track` flags for that purpose.
